### ISSUE TITLE
Amazon Timestream APIs missing from AWS PowerShell module?

### ISSUE THREAD

**Issue by pcgeek86** [aws/aws-tools-for-powershell#201]
## Description

I cannot find the Amazon Timestream APIs in the AWS Tools for PowerShell. The service documentation, including the API Reference, is located here: https://docs.aws.amazon.com/timestream/latest/developerguide/what-is-timestream.html

## Reproduction Steps

```
C:\> docker run --rm -it microsoft/powershell
PowerShell 7.1.2
Copyright (c) Microsoft Corporation.

https://aka.ms/powershell
Type 'help' to get help.

PS /> Install-Module -Name awspowershell.netcore -Force
PS /> gcm -mod *times*
PS /> gcm -mod *time*
PS /> ipmo AWSPowerShell.NetCore
PS /> Get-AWSCmdletName -ApiOperation createdatabase

CmdletName       ServiceOperation ServiceName
----------       ---------------- -----------
New-GLUEDatabase CreateDatabase   AWS Glue                                                      
```

---

This is a :bug: bug-report

**Comment by ashishdhingra** [comment]
Hi @pcgeek86,

Good morning.

I had discussion with development team, they are working on to include the TimeStream API in AWS .NET SDK and PowerShell. We would post any updates here as the progress is made.

Thanks,
Ashish

**Comment by ashovlin** [comment]
Timestream Query and Write are now included in the 4.1.12.0 release, either in their own modules or the full AWSPowerShell or AWSPowerShell.NetCore.

1.	https://www.powershellgallery.com/packages/AWS.Tools.TimestreamQuery/4.1.12.0
2.	https://www.powershellgallery.com/packages/AWS.Tools.TimestreamWrite/4.1.12.0
